advanced/am-pattern-4/README.md

```diff
@@ -0,0 +1,304 @@
+| `wso2.deployment.am.gateway.replicas`                                       | Number of replicas of API Manager Gateway to be started                                   | 2                           |
+| `wso2.deployment.am.gateway.strategy.rollingUpdate.maxSurge`                | Refer to [doc](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#deploymentstrategy-v1-apps)  | 2                         |
+| `wso2.deployment.am.gateway.strategy.rollingUpdate.maxUnavailable`          | Refer to [doc](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#deploymentstrategy-v1-apps)  | 0                         |
+| `wso2.deployment.am.gateway.config`                                         | Custom deployment configuration file for Gateway profile (`<WSO2AM>/repository/conf/deployment.toml`)         | -                           |
+| `wso2.deployment.am.km.livenessProbe.initialDelaySeconds`                   | Initial delay for the live-ness probe for API Manager Key Manager                             | 60                          |
+| `wso2.deployment.am.km.livenessProbe.periodSeconds`                         | Period of the live-ness probe for API Manager Key Manager                                     | 10                          |
+| `wso2.deployment.am.km.readinessProbe.initialDelaySeconds`                  | Initial delay for the readiness probe for API Manager Key Manager                             | 60                          |
+| `wso2.deployment.am.km.readinessProbe.periodSeconds`                        | Period of the readiness probe for API Manager Key Manager                                     | 10                          |
+| `wso2.deployment.am.km.replicas`                                            | Number of replicas of API Manager Key Manager to be started                                   | 2                           |
+| `wso2.deployment.am.km.strategy.rollingUpdate.maxSurge`                     | Refer to [doc](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#deploymentstrategy-v1-apps)  | 2                         |
+| `wso2.deployment.am.km.strategy.rollingUpdate.maxUnavailable`               | Refer to [doc](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#deploymentstrategy-v1-apps)  | 0                         |
+| `wso2.deployment.am.km.config`                                              | Custom deployment configuration file for Key Manager profile (`<WSO2AM>/repository/conf/deployment.toml`)         | -                           |
+| `wso2.deployment.am.pubDevPortalTM.hostname`                                | Hostname for API Manager Publisher, DevPortal and Carbon Management Console               | `am.wso2.com`               |
```

Comment:
@chathuraswso2 where can I find the property details of Ingress for APIM? If missing, please add them.